fluent-plugin-parser-maybejson.gemspec

```
@@ -4,7 +4,7 @@ $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
 Gem::Specification.new do |spec|
   spec.name          = "fluent-plugin-parser-maybejson"
-  spec.version       = "0.1.1"
+  spec.version       = "0.1.2"
```

```suggestion
  spec.version       = "1.0.0"
```

Please do a major version update since `fluentd` is also being updated by a major version.

---

done